train.py

```diff
@@ -682,7 +745,9 @@ def main():
 def train_one_epoch(
         epoch, model, loader, optimizer, loss_fn, args,
         lr_scheduler=None, saver=None, output_dir=None, amp_autocast=suppress,
-        loss_scaler=None, model_ema=None, mixup_fn=None):
+        loss_scaler=None, model_ema=None, mixup_fn=None,
+        cuda_graph=None, cg_stage=None,
```

Comment:
Can you add some comments about the newly added arguments. Explicitly what they are, and why they're important. You can reference the cuda graphs docs if that makes it easier.